airbyte-integrations/connectors/source-gainsight-px/source_gainsight_px/manifest.yaml

```diff
@@ -0,0 +1,142 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: []
+
+  custom_selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["{{ parameters.extractorPath }}"]
+
+  requester:
+    type: HttpRequester
+    url_base: "https://api.aptrinsic.com/v1/"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "X-APTRINSIC-API-KEY"
+      api_token: "{{ config['api_key'] }}"
+
+  retriever:
+    type: SimpleRetriever
+    paginator:
```

Comment:
The connector allows to use pagination
![image](https://github.com/airbytehq/airbyte/assets/5154322/86709b20-3a1a-44ab-a0fd-b7d4fcc1ecdd)
Please implement it.

Reply:
CursorPagination Implemented